services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/client/gRPC/GoalStateClientImpl.java

```diff
@@ -170,17 +231,26 @@ public void onCompleted() {
 
         StreamObserver<Goalstate.GoalStateV2> requestObserver = asyncStub.pushGoalStatesStream(responseObserver);
         try {
-                Goalstate.GoalStateV2 goalState = hostGoalState.getGoalState();
-                logger.log(Level.INFO, "Sending GS to Host " + hostIp + " as follows | " + goalState.toString());
-
-                requestObserver.onNext(goalState);
+            Goalstate.GoalStateV2 goalState = hostGoalState.getGoalState();
+            logger.log(Level.INFO, "Sending GS to Host " + hostIp + " as follows | " + goalState.toString());
+            requestObserver.onNext(goalState);
+            if (hostGoalState.getGoalState().getNeighborStatesCount() == 1 && (hostIp.equals("192.168.20.92") || hostIp.equals("10.213.43.92"))) {
```

Comment:
This is internal test IPs and we should get rid of them all in the master codes.

Reply:
Removed those IPs from the code, and created a varaible called `monitorHosts` in the java code, which comes from a variable called `grpc.monitor-hosts` in `application.properties`.